inst/htmljs/animint.js

```diff
@@ -2068,14 +2072,14 @@ if ('col' in response.plots[p_name] && !('row'in response.plots[p_name])){
       maximum_column=response.plots[p_name].col
     }
   }
-  
-  var construct_outer_table = function () {
+   }
+  find_maximum_dimensions()
     var count_dimensions = Math.ceil(Math.sqrt(count))-1;
     var outer_table = element.append("table").attr("id", "outerTable").style("width", "1500px").style("border", "1px solid white");
     maximum_row=Math.max(maximum_row,count_dimensions)
```

Comment:
this is probably the global variable which is causing the test to fail

Reply:
one of them

Reply:
yes. I had realized the reason why global variables test was failing. one of the variables did not have any prefix like var or let. so it was implicitly global